Source/WebCore/layout/formattingContexts/inline/InlineItemsBuilder.h

```diff
@@ -72,6 +72,7 @@ class InlineItemsBuilder {
     bool m_contentRequiresVisualReordering { false };
     bool m_isTextAndForcedLineBreakOnlyContent { true };
     size_t m_inlineBoxCount { 0 };
+    bool m_hasSeenTextAutospace { !root().style().textAutospace().isNoAutospace() };
```

Comment:
should this always be false? at construction time we certainly have not seen any content with autospace (unless this member actually means something slightly different)